car/gocar.go

```diff
@@ -1,4 +1,4 @@
-// A car exporter based on go-car module
+// A car exporter based on boxo/ipld/car module
```

Comment:
why did we adopt `boxo`?

Reply:
ipfs moved a bunch of its code in a separate repo called boxo, it was just some refactoring from their end. The kubo repo now depends on boxo too, just like we need to now.